pkg/scheduler/preemption/preemption_test.go

```diff
@@ -91,6 +91,23 @@ func TestPreemption(t *testing.T) {
 				Resource(corev1.ResourceCPU, "6", "6").
 				Resource(corev1.ResourceMemory, "3Gi", "3Gi").
 				Obj(),
+				*utiltesting.MakeFlavorQuotas("alpha").
```

Comment:
Instead, change the test to use `default` for both resources. They are part of the same resource group, so they shouldn't get different flavors.